docs/getting-started-in-5-minutes.md

```diff
@@ -80,32 +88,63 @@ authors = ["Your Name"]
 edition = "2021"
 license = "GPL-3.0"
 
-[lib]
-crate-type = ["cdylib"]
-
 [dependencies]
 gstd = { git = "https://github.com/gear-tech/gear.git", features = ["debug"] }
 
-[profile.release]
-lto = true
-opt-level = 's'
+[build-dependencies]
+gear-wasm-builder = { git = "https://github.com/gear-tech/gear.git" }
+
+[dev-dependencies]
+gtest = { git = "https://github.com/gear-tech/gear.git" }
 ```
 
 5. Replace the default contents of `lib.rs` with the code for our first smart-contract. Open `src/lib.rs` in your editor and paste the following code:
 
 ```rust
 #![no_std]
 
-use gstd::{msg, prelude::*};
+use gstd::{debug, msg, prelude::*};
+
+static mut MESSAGE_LOG: Vec<String> = vec![];
 
 #[no_mangle]
 pub unsafe extern "C" fn handle() {
     let new_msg = String::from_utf8(msg::load_bytes()).expect("Invalid message");
 
     if new_msg == "PING" {
-        msg::reply_bytes("PONG", 12_000_000, 0);
+        msg::reply_bytes("PONG", 0);
     }
 
+    MESSAGE_LOG.push(new_msg);
+
+    debug!("{:?} total message(s) stored: ", MESSAGE_LOG.len());
+
+    for log in MESSAGE_LOG.iter() {
+        debug!(log);
+    }
+}
+
+/// and a simple unit test:
+
+#[cfg(test)]
+mod tests {
+    extern crate std;
+
+    use gtest::{Log, Program, System};
+
+    #[test]
+    fn it_works() {
+        let system = System::new();
+        system.init_logger();
+
+        let program = Program::current(&system);
+
+        let res = program.send_bytes(42, "INIT");
+        assert!(res.log().is_empty());
+
+        let res = program.send_bytes(42, "PING");
+        assert!(res.contains(&Log::builder().source(1).dest(42).payload_bytes("PONG")));
```

Comment:
I would separate `Log` build into some var to increase the readability

Reply:
ok, removed from getting started. in examples will be updated after change in /apps/
https://github.com/gear-tech/apps/blob/master/ping/src/lib.rs